.github/workflows/build.yml

```diff
@@ -38,25 +38,29 @@ jobs:
           node-version: ${{ env.NODE_VERSION }}
           cache: yarn
 
-      - name: Installing dependencies
-        run: yarn install --frozen-lockfile
-
-      - name: Build ${{ env.NPM_VERSION }}
-        run: yarn build
-
-      - name: semantic-release
-        run: echo "Not pushing to npm yet"
-
-        # run: |
-        #   echo '//registry.npmjs.org/:_authToken=${NPM_TOKEN}' >> ./.npmrc
-        #   npx semantic-release --dry-run ${{env.DRY_RUN}}
-        #   git checkout -- .npmrc
-        # env:
-        #   GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
-        #   NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-
-      # - name: Upload docs
-      #   uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # tag=v3.1.0
-      #   with:
-      #     name: docs
-      #     path: tmp/docs/
+      - name: set git config
+        run: |
+          git config --global user.email "you@example.com"
+          git config --global user.name "Your Name"
+
+      - name: Clone docker-renovate
+        run: git clone https://github.com/renovatebot/docker-renovate
+
+      - name: fix Dockerfile
+        run: |
+          sed -i 's|yarn version --new-version ${RENOVATE_VERSION}|yarn version --new-version ${RENOVATE_VERSION} --no-git-tag-version|' docker-renovate/Dockerfile
+          sed -i 's|COPY bin/ /usr/local/bin/||' docker-renovate/Dockerfile
+
+      - name: Docker build
+        run: docker build . -f docker-renovate/Dockerfile -t renovate:slim
+
+      - name: Docker tag 
+        run: docker tag renovate:slim artifactory.netskope.io/ep-tools-docker/instruments/renovate:slim
+      
+      - name: Docker push
+        run: |
+          docker login --username ${NS_ARTIFACTORY_USER} --password ${NS_ARTIFACTORY_PASSWORD} artifactory.netskope.io
+          docker push -a artifactory.netskope.io/ep-tools-docker/instruments/renovate
```

Comment:
why this instead `docker push artifactory.netskope.io/ep-tools-docker/instruments/renovate:slim`

Reply:
yes, we could do that because there are no other tags to push